### PR TITLE
Documentation updates for correctness and demonstration 

### PR DESCRIPTION
**Pull request by masoncusack**
Hey! ✌️

Made the following changes to update the docs and smooth adoption for people, especially those hoping to use their deployed resources for Python notebooks, and deploying clusters to Azure:

- Update `spark_version` cluster resource parameter to "required"
- Update `node_type_id` cluster resource parameter to "required"
- Remove JUPYTER from the documented notebook format types. It was rejected when I used it, but SOURCE worked just fine for Python.
- Add example to notebook documentation for quick deployment of an empty Python notebook (common use case)
- Note that "absolute path" to deployed notebooks is not a regular file system path, but must start specifically with "/", giving an example for a notebook.

### CODE REVIEW

File: website/content/Resources/cluster.md [hunk 1]
```diff
@@ -144,7 +146,7 @@ value must be within the range 500 - 4096. Custom EBS volumes cannot be specifie
 is optional; if unset, the driver node type will be set as the same value as node_type_id defined above.
 
 #### - `node_type_id`:
-> **(Optional)** This field encodes, through a single value, the resources 
+> **(Required)** This field encodes, through a single value, the resources
```
[the remark this answers is not in the document]
Comment (masoncusack) [reply website/content/Resources/cluster.md]: Ok, I've made a docs change to clarify the dependency between these two, added to the schemas as you suggested.

File: website/content/Resources/notebook.md [hunk 2]
```diff
@@ -51,7 +64,7 @@ returns an error RESOURCE_ALREADY_EXISTS. If this operation fails it may have su
 
 #### - `format`:
 > **(Required)** This specifies the format of the file to be imported. 
-By default, this is SOURCE. However it may be one of: SOURCE, HTML, JUPYTER, DBC. The value is case sensitive.
```
[the remark this answers is not in the document]
Comment (masoncusack) [reply website/content/Resources/notebook.md]: Done. :) Out of curiosity, what exactly is the difference between Source and Jupyter when the language is set to Python?

I may raise a separate issue for this kind of information to be documented in future.

[the remark this answers is not in the document]
Comment (masoncusack) [reply website/content/Resources/notebook.md:54]: Added a comment on the utility of using Jupyter for deploying notebooks to Azure workspaces, linking to the Azure doc you provided.

File: website/content/Resources/notebook.md [hunk 3]
```diff
@@ -51,7 +64,7 @@ returns an error RESOURCE_ALREADY_EXISTS. If this operation fails it may have su
...
+By default, this is SOURCE. However it may be one of: SOURCE, HTML, JUPYTER, DBC. The value is case sensitive. Note: For Azure, JUPYTER differs from other options in producing a notebook in a file format supported by the [workspace import and export API](https://docs.microsoft.com/en-us/azure/databricks/dev-tools/api/latest/workspace#--request-structure-1).
```
Comment (stikkireddy) [inline website/content/Resources/notebook.md:67]: Hey can you elaborate more on that statement. I do not quite understand it. I believe regardless of the cloud, the JUPYTER option works, it just requires the input notebooks to be in a .ipynb format. I think the statement should be more in regards to stating that, when using the JUPYTER format, the notebooks should be in the .ipynb extension, SOURCE should be either .scala, .py, .r, .sql extension based files, DBC should be extension .dbc, and HTML is .html. 

Please let me know if you have any questions.

Comment (masoncusack) [reply website/content/Resources/notebook.md:67]: Hey, the documentation you linked was for an Azure API, so I specified this. I'll make the suggested update and remove the cloud-specific reference.

**Comment by stikkireddy** [comment]
hey @masoncusack thanks for the contribution!

**Pull request merged**